src/xls_handler.py

```diff
@@ -163,31 +164,35 @@ def get_day_timetable(
         :param table_dict: dict with parsed groups and their schedule
         :param day_index: number of day in a week
         :param week_index: academic week index
-        :param format: `bool` default True
+        :param formatting: `bool` default True
         :param mode: `bool` change header order. Default False
         :return: string with parsed markdown-like style data
         """
         is_empty = False
 
         if mode is False:
-            res = f"*{self.day_of_week[str(day_index)]} {week_index}*\n" \
-                  f"*-------------------------*\n"
+            res = (
+                f"*{self.day_of_week[str(day_index)]} {week_index}*\n"
+                f"*-------------------------*\n"
+            )
         else:
-            res = f"*-------------------------*\n" \
-                  f"*{self.day_of_week[str(day_index)]}:*\n"
+            res = (
+                f"*-------------------------*\n"
+                f"*{self.day_of_week[str(day_index)]}:*\n"
+            )
         group_dict = table_dict[group]
         week_dict = group_dict[f"week: {week_index}"]
         day_dict = week_dict[f"day: {day_index}"]
 
         for lesson in range(1, 6):
             value = day_dict[f"{lesson} lesson: "]
-            if value == "":
+            if not value:
                 value = "_----_"
             res += f"*{lesson})* {value}\n"
 
         res = res[:-1]
 
-        if format is True:
+        if formatting is True:
```

Comment:
If formatting can be only either True or False it's better to leave only `if formatting`